lib/src/definitions.dart

```diff
@@ -263,11 +263,46 @@ class OnExternalLinkClickedResult {
   });
 }
 
+class PoiFilter {
+  /// A [text] filter that will
```

Comment:
Coidado con este texto, hai algunha movida gramatical:

/// A text filter that will filter the search results and POIs,
/// displaying only the ones that match.

Tamén: por que enlazas text nadoc do propio text?

Reply:
Fixeado